extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -180,10 +180,13 @@ class AmpVideoIframe extends AMP.BaseElement {
    */
   getMetadata_() {
     const {sourceUrl, canonicalUrl} = Services.documentInfoForDoc(this.element);
+    const rootNode = this.getAmpdoc().getRootNode();
 
     return dict({
       'sourceUrl': sourceUrl,
       'canonicalUrl': canonicalUrl,
+      'title': this.getAmpdoc().getRootNode().title || null,
+      'lang': rootNode.documentElement ? rootNode.documentElement.lang : null,
```

Comment:
Cannot recall if we landed transpilation for optional chaining.

`'lang': rootNode.documentElement?.lang || null`